README.md

```diff
@@ -1196,6 +1197,21 @@ module.exports = {
 }
 ```
 
+#### Content Security Policy
+
+Next.js supports [CSP](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP). You can set your CSP Poliy in your Next.js configuration.
```

Comment:
I think there's no need to `Policy` after CSP.

Reply:
https://github.com/zeit/next.js/pull/4943#issuecomment-424948950